packages/ui/src/components/TextLink/index.tsx

```diff
@@ -1,10 +1,10 @@
-import React, { ReactChildren } from 'react';
+import React, { ReactChild } from 'react';
 import classNames from 'classnames';
 import styles from './index.module.scss';
 
 export type Props = {
   className?: string;
-  children: ReactChildren;
+  children: ReactChild;
```

Comment:
Should we support `<b>some text</b> and else`

Reply:
most likely it'll be pure string. since we need standardize those in component's props instead of using html tags. waiting for a better design system.